md_to_github.py

```diff
@@ -53,6 +53,11 @@ def toc_md_files(toc, indices):
 -->
 '''
 
+acknowledgement = r'''
+***
+<sub>\* Other names and brands may be claimed as the property of others.</sub>
+'''
+
```

Comment:
This should not be part of the md_to_github script: The md sources are compiled into both simics-native docs and into github wiki, so better fix this in the md sources instead. Or is this stuff also auto-added by dodoc?

Reply:
The Simics Online Documentation will have a separate solution that won't require authors to manually add acknowledgment lines. So this DML wiki exclusive logic should indeed be DML wiki exclusive.

Reply:
Alternatively... the DML wiki could have a `* Legal Information` page... hmm...